esrally/utils/progress.py

```diff
@@ -1,14 +1,18 @@
 import sys
+import os
 
 
 class CmdLineProgressReporter:
     """
     CmdLineProgressReporter supports displaying an updating progress indication together with an information message.
     """
 
-    def __init__(self, width=80):
+    def __init__(self, width=80, plain_output=False):
         self._width = width
         self._first_print = True
+        self._plain_output = plain_output
+        if (os.environ['TERM'] == 'dumb'):
```

Comment:
Good idea, I pushed a change to use `.get` which will return `None` so this doesn't throw a `KeyError`